### PR TITLE
779/refactor: Request and response in verbose mode

### PR DESCRIPTION
**Pull request by herenacreative**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. 779/refactor: Request in verbose mode

## How to test  
1. bin/run --verbose

 send request:
Screen Shot 2022-08-18 at 09 18 17

### CODE REVIEW

**Comment by sapiderman** [comment]
@herenacreative 
Ok... im getting the request.. good.
Still not getting response though

image

**Review by nicnocquee — CHANGES_REQUESTED** [review]
In verbose mode, user should be able to see what request was sent by Monika **every time it sends the request**, which includes the method, the URL, and the header (if any).

Seems like this PR doesn't show that information.

Screenshot 2022-08-10 at 09 47 56

**Comment by nicnocquee** [comment]
@herenacreative still need to update the probes config, especially need to format it so that it's clear that there are multiple requests in a probe.

Currently,

```
Probes:
- Probe ID: 1
    Name: GitHub
    Description: Multiple
    Interval: 20
    Request Method: GET
    Request URL: https://github.com
    Request Headers: -
    Request Body: -
    Request Method: GET
    Request URL: https://github.com/hyperjump
    Request Headers: -
    Request Body: -
    Alerts: [{ "query": "response.status < 200 or response.status > 299", "message": "HTTP Status is not 200"},
            { "query": "response.time > 2000", "message": "Response time is more than 2000ms" }]
```

It's better to be,

```
Probes:
- Probe ID: 1
    Name: GitHub
    Description: Multiple
    Interval: 20
    Requests:
      - Request Method: GET
         Request URL: https://github.com
         Request Headers: -
         Request Body: -
      - Request Method: GET
         Request URL: https://github.com/hyperjump
         Request Headers: -
         Request Body: -
    Alerts: [{ "query": "response.status < 200 or response.status > 299", "message": "HTTP Status is not 200"},
            { "query": "response.time > 2000", "message": "Response time is more than 2000ms" }]
```

Also, format the alerts so that it doesn't look like JSON.

Basically, they should look like the YAML format.

**Review by nicnocquee — APPROVED** [review]

File: src/components/logger/request-log.ts [hunk 1]
```diff
@@ -103,23 +105,28 @@ export class RequestLog {
...
+        this.response?.alive ? 'alive' : 'dead'
+      } ${this.request?.url} ${
+        this.response?.alive ? this.response?.responseTime : '-'
+      }ms packetLoss:${this.response?.alive ? this.response?.packetLoss : '-'}%`
+    } else {
+      probeMsg = flags.verbose
+        ? `${probeMsg} 
+      Request Headers:
+      ${JSON.stringify(this.request?.headers) || '-'} 
+      Request Body:
```
Comment (sapiderman) [inline src/components/logger/request-log.ts:124]: Suggestion:  Put hese in one line, more compact and easier to read imho
```javascript
Request Headers: $(JSON.stringify(this.request?headers) || '-'}
```

Then the key and value are in one line:
image

vs

image

**Review by sapiderman — APPROVED** [review]
Small suggestion but looks ok otherwise. Good work!

**Pull request merged**